generated/nidcpower/nidcpower/session.py

```diff
@@ -4846,11 +4862,27 @@ def measure_multiple(self):
 
         '''
         import collections
-        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance'])
+        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance', 'channel'])
 
         voltage_measurements, current_measurements = self._measure_multiple()
 
-        return [Measurement(voltage=voltage_measurements[i], current=current_measurements[i], in_compliance=None) for i in range(self._parse_channel_count())]
+        if self._repeated_capability == '':
+            channel_names = self._get_channel_names(range(self.channel_count))
+        else:
+            first_channel_name = self._get_channel_name(1)
+            channel_names = _converters.convert_channels_repeated_capabilities(
+                self._repeated_capability,
+                first_channel_name
+            )
+
+        return [
+            Measurement(
+                voltage=voltage_measurements[i],
+                current=current_measurements[i],
+                in_compliance=None,
+                channel=channel_names[i],
+            ) for i in range(self._parse_channel_count())
```

Comment:
Is it true that the result of _parse_channel_count must be the same as the length of `channel_names`?
I wonder if there are  more pythonic ways of doing this (zip?) that don't require calling into the driver runtime again.